src/make-http-request.coffee

```diff
@@ -43,9 +43,7 @@ makeHTTPRequest = (method, url, data, headers = {}, modify) ->
       if error?.status is 408
         resolve makeHTTPRequest.apply null, originalArguments
```

Comment:
I agree that JSON API client should be low level and not make decisions on how to handle responses. My concern with moving it to the Panoptes JS client is if we move this handler to the Panoptes JS client and if we decide to merge the JSON API client and Panoptes JS client, then that separation of concerns wouldn't be present anymore. With that in mind, moving this handler to PFE or moving it to the Panoptes JS client and keeping the clients separate would be best. If moved to PFE, I also don't want us spending a lot of time adding this kind of handling to most requests since I think it might be likely we would want this behavior most of the time. What do you think the best approach here is?